src/AltinnCore/Testdata/Party/10004/party.json

```diff
@@ -1,13 +1,13 @@
-﻿{
-	"partyId": 10004,
-	"partyTypeName": 2,
-	"orgNumber": "900000004",
-	"ssn": null,
-	"unitType": "BEDR",
-	"name": "Frisk & Sporty Trondheim",
-	"isDeleted": false,
-	"onlyHierarchyElementWithNoAccess": false,
-	"person": null,
-	"organization": null,
-	"childParties": null
-}
\ No newline at end of file
+{
+  "partyId": 10004,
+  "partyTypeName": 2,
+  "orgNumber": "900000004",
+  "ssn": null,
+  "unitType": "BEDR",
+  "name": "Frisk & Sporty Trondheim",
+  "isDeleted": false,
+  "onlyHierarchyElementWithNoAccess": false,
+  "person": null,
+  "organization": null,
```

Comment:
maybe add the org so the testdata set is complete ?

Reply:
The orgs are populated from the /orgs/ folder :) So no need to declare them two places.